wandb/sdk/launch/_project_spec.py

```diff
@@ -147,6 +147,9 @@ def _fetch_project_local(self, api: Api) -> None:
                 self.project_dir, run_info["git"]["remote"], run_info["git"]["commit"]
             )
             patch = utils.fetch_project_diff(self.uri, api)
+            if run_info.get("python"):
+                with open(os.path.join(self.project_dir, "runtime.txt"), "w") as fp:
```

Comment:
do you want this in aux_dir instead, i thought project_dir was just for code

Reply:
ah, you're right. I'll follow a similar pattern to the run_config pattern I proposed in another comment.